ticket_custom_fields.view.lkml

```diff
@@ -0,0 +1,176 @@
+view: ticket_custom_fields {
+  derived_table: {
+    sql:
+      SELECT DISTINCT
+        tickets__fields._sdc_source_key_id AS ticket_id,
+
+        MAX(tickets__fields.value) FILTER (WHERE tickets__fields.id = 21140580)
```

Comment:
We could add a couple comments regarding the hard-coded numeric values, although it's somewhat self-explanatory..